crates/bevy_remote/src/lib.rs

```diff
@@ -0,0 +1,877 @@
+//! The Bevy Remote Protocol, an HTTP- and JSON-based protocol that allows for
```

Comment:
nit: if i understand correctly, the protocol itself is transport and serialization-agnostic, and this is just one implementation for this combination of transport/serde?

Reply:
I reworded the comment.